_data/projects/NodeCG.yml

```diff
@@ -0,0 +1,12 @@
+name: NodeCG
+desc: Create broadcast graphics using Node.js and a browser
+site: https://nodecg.com/
```

Comment:
There's an invalid cert on this site (because it's hosted on GitHub Pages) - does it need to be `https`?